src/seedu/addressbook/commands/Command.java

```diff
@@ -41,6 +41,13 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
      */
     public abstract CommandResult execute();
 
+    
+    /**
+     * Returns true if command mutates the data
```

Comment:
It is better to mention it returns true if the Command *potentially* mutates the data (header comments specify a contract, so try to be as precise as possible). E.g. Adding a duplicate person will not mutate data but the corresponding AddCommand object will still return true for this method.